11-Backbonejs/events.md

```diff
@@ -29,12 +29,8 @@ Once this code has been run, whenever our `bookList` emits a `update` event the
 
 Why is this important? **It means we can separate code that updates the model from code that updates the DOM.** In practice, this allows us to simplify and DRY our code that handles DOM events.
 
-TODO DPR: diagram. Something like
-```
-     => DOM event handler (jQuery) =>
-DOM                                    Model / Collection
-     <= Backbone event handler     <=
-```
+![backbone-event-cycle](images/model-event-diagram-generic.png)
```

Comment:
Do we have the resource from which this diagram was created when/if we need to edit? I like to add them as a comment after the image

Reply:
Chris did the work, I have asked him for it.